users/permissions.py

```diff
@@ -14,4 +14,8 @@ def has_permission(self, request, view):
             return True
         return request.user and request.user.is_authenticated and request.user.user_type == 'Admin'
     def has_object_permission(self, request, view, obj):
-        return obj == request.user or request.user.user_type == 'Admin'
+        if request.user:
+            if request.user.user_type == 'Admin':
```

Comment:
Can be:
`return request.user.user_type == 'Admin' or obj == request.user`

Reply:
Done

Reply:
Done